sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -57,8 +57,9 @@ static void report_test_status(bool result) {
   // Reinitialize UART before print any debug output if the test clobbered it.
   if (kDeviceType != kDeviceSimDV && kTestConfig.can_clobber_uart) {
     init_uart();
-    test_coverage_send_buffer();
   }
+  LOG_INFO("Finished test %s", kTestConfig.file);
+  test_coverage_send_buffer();
```

Comment:
Is it ok to do this unconditionally? What happens if we send the buffer without init?

Reply:
This is a NOP now since we don't have on-target coverage enabled. `init_uart()` will be called if needed but we should send this buffer unconditionally not just if `can_clobber_uart`.